File: BR-API/BR-API/Entities/Vendedor.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace BR_API.Entities
{
    public class Vendedor
    {
        public int Id { get; set; }
        public string Nombre { get; set; }
        public string Apellido { get; set; }
        public string Telefono { get; set; }
        public Propiedad Propiedad { get; set; }
    }
}

File: BR-UI/src/app/vendedores/vendedor.ts
export interface Vendedor {
    id: number
    nombre: string
    apellido: string
    telefono: string
    propiedadId: number
    propiedadTitulo: string
}

export interface VendedorDTO {
    id: number
    nombre: string
    apellido: string
    telefono: string
}

export interface VendedorCreacionDTO {
    nombre: string
    apellido: string
    telefono: string
}
